internal/cmd/root.go

```diff
@@ -42,6 +42,7 @@ operators from the installed catalogs.`,
 		newOperatorListAvailableCmd(&cfg),
 		newOperatorDescribeCmd(&cfg),
 		newVersionCmd(),
+		newOperatorListCustomResourcesCmd(&cfg),
```

Comment:
Move this up to group it with the other operator subcommands. How about between ListAvailable and Describe?